lua/orgmode/org/mappings.lua

```diff
@@ -295,6 +295,31 @@ function OrgMappings:todo_prev_state()
   self:_todo_change_state('prev')
 end
 
+function OrgMappings:toggle_heading()
+  local line = vim.fn.getline('.')
+  local parent = Files.get_closest_headline()
```

Comment:
This does not work on lines without a parent headline. However, Emacs Orgmode will turn such a line into a top-level headline.